From c9307bfb1cb885b858b83c511369ac089bda4ada Mon Sep 17 00:00:00 2001
From: Rui Matos <tiagomatos@gmail.com>
Date: Tue, 21 Mar 2017 18:56:04 +0100
Subject: [PATCH 2/4] data: Start gnome-shell in the DisplayServer autostart
 phase

This is what gnome-shell itself does nowadays and it's required for
gnome-settings-daemon which starts in the Initialization phase to not
block waiting for gnome-shell which would in turn prevent gnome-shell
to start and thus deadlock the session.

https://bugzilla.gnome.org/show_bug.cgi?id=780362
---
 data/setup-shell.desktop | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/setup-shell.desktop b/data/setup-shell.desktop
index 7fa1e7f..cce175e 100644
--- a/data/setup-shell.desktop
+++ b/data/setup-shell.desktop
@@ -3,6 +3,6 @@ Type=Application
 Name=GNOME Shell
 Exec=gnome-shell --mode=initial-setup
 X-GNOME-AutoRestart=true
-X-GNOME-Autostart-Phase=WindowManager
+X-GNOME-Autostart-Phase=DisplayServer
 X-GNOME-Provides=panel;windowmanager;
 X-GNOME-Autostart-Notify=true
-- 
2.13.6

